policy:
  array_based: true
  n_points: 4
  spin_dist: 22.5